apps/orgs/utils.py

```diff
@@ -39,14 +39,14 @@ def get_org_from_request(request):
     if org and org.internal:
         # 内置组织直接返回
         return org
-    
+
     if not settings.XPACK_ENABLED:
         # 社区版用户只能使用默认组织
         return Organization.default()
-    
+
     if not org and request.user.is_authenticated:
         # 企业版用户优先从自己有权限的组织中获取
-        org = request.user.orgs.first()
+        org = request.user.orgs.exclude(id=Organization.SYSTEM_ID).first()
 
     if not org:
         org = Organization.default()
```

Comment:
The code seems fine as no errors were found, and there didn't seem to be any unnecessary parts or redundancies. No optimizations required here.

It's worth noting that `settings.XPACK_ENABLED` is only used after setting up an organization. It's generally recommended against directly reading global variables (like session state) into functions since they could expose sensitive user information. However, this approach avoids having to create a new variable inside the context of accessing settings which can lead to confusion.